keyboards/ciaanh/kanagawa/readme.md

```diff
@@ -0,0 +1,23 @@
+# Kanagawa
```

Comment:
This should follow the [template](https://github.com/qmk/qmk_firmware/blob/master/data/templates/keyboard/readme.md) closer.